docs/quickstart.html

```diff
@@ -304,15 +304,24 @@
 </p>
 
 <pre>
-&gt; <b>cat /tmp/file-input.txt | ./bin/kafka-console-producer --broker-list localhost:9092 --topic streams-file-input</b>
+&gt; <b>bin/kafka-topics.sh --create \</b>
```

Comment:
@guozhangwang Strange, it seems to look fine in my browser.. wouldn't double-quotes look a bit strange, given that in other similar cases the keywords are in bold?